[neeo] ops4j webservice duplicate servlet

Neeo started throwing this after the karaf update when OH starts.  The binding looks to work otherwise.  To note I've updated to SNAPSHOT 3347 to update beyond the JuPnP upgrade.

2023-03-05 12:30:13.342 [ERROR] [org.openhab.io.neeo.NeeoService     ] - Error during servlet startup
org.osgi.service.http.NamespaceException: ServletModel{id=ServletModel-35,name='org.openhab.io.neeo.internal.NeeoBrainServlet',alias='/neeo/NEEO-05dbea22',urlPatterns=[/neeo/NEEO-05dbea22/*],servlet=org.openhab.io.neeo.internal.NeeoBrainServlet@597ccd7b,contexts=[{HS,OCM-31,default,/}]} can't be registered. ServletContextModel{id=ServletContextModel-3,contextPath='/'} already contains servlet named org.openhab.io.neeo.internal.NeeoBrainServlet: ServletModel{id=ServletModel-34,name='org.openhab.io.neeo.internal.NeeoBrainServlet',alias='/neeo/NEEO-713af1e8',urlPatterns=[/neeo/NEEO-713af1e8/*],servlet=org.openhab.io.neeo.internal.NeeoBrainServlet@aab7de4,contexts=[{HS,OCM-31,default,/}]}
        at org.ops4j.pax.web.service.spi.model.ServerModel.addServletModel(ServerModel.java:1458) ~[?:?]
        at org.ops4j.pax.web.service.internal.HttpServiceEnabled.lambda$doRegisterServlet$1(HttpServiceEnabled.java:477) ~[?:?]
        at org.ops4j.pax.web.service.spi.model.ServerModel.lambda$run$2(ServerModel.java:541) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]

--- a/bundles/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/NeeoService.java
+++ b/bundles/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/NeeoService.java
@@ -282,7 +282,10 @@ public class NeeoService implements EventSubscriber, NetworkAddressChangeListene
                             sysInfo.getHostname(), ipAddress, clientBuilder);
                     servlets.add(newServlet);
 
-                    localContext.getHttpService().registerServlet(servletUrl, newServlet, new Hashtable<>(),
+                    Hashtable<Object, Object> initParams = new Hashtable<>();
+                    initParams.put("servlet-name", servletUrl);
+
+                    localContext.getHttpService().registerServlet(servletUrl, newServlet, initParams,
                             localContext.getHttpService().createDefaultHttpContext());
                     logger.debug("Started NEEO Listener at {}", servletUrl);
                 } catch (NamespaceException | ServletException | IOException e) {

